src/kernl/implementations/layer_norm.py

```diff
@@ -59,16 +59,16 @@ def pytorch_naive_rmsnorm(a: torch.Tensor, weight: torch.Tensor, eps: float):
 
 @triton.jit
 def layer_norm_xformers(
-    Out,
-    A,
-    Weight,
-    Bias,
-    Mean,
-    Rstd,
-    stride_row_out,
-    stride_col_out,
-    stride_row_a,
-    stride_col_a,
+    output_ptr,
+    a_ptr,
+    weight_ptr,
+    bias_ptr,
+    mean_ptr,
+    rstd_ptr,
+    output_row_stride,
+    output_col_stride,
+    a_row_stride,
+    a_col_stride,
     N,
```

Comment:
should we use `n_size` and `block_n_size` ?

Reply:
done